### PR TITLE
#163020921 Improved UI design, refactored code and folder structure

### PR DESCRIPTION
**Pull request by chikeozulumba**
#### What does this PR do?
Implement feedback gotten from LFA
#### Description of Task to be completed?
- Refactored UI Pages
- Added page for creating meetups
- Renamed Server folder
- Updated the build scripts
- Restructured Routes
- Refactored Code
- Implement Register redirect to profile
- Restructure Assets folder
- Update README.md
- Changed `postinstall` to `heroku-postbuild`
#### What are the relevant pivotal tracker stories?
[#163020921](https://www.pivotaltracker.com/story/show/163020921)

### CODE REVIEW

File: UI/pages/login.html [hunk 1]
```diff
@@ -91,7 +91,7 @@ <h3 class="md-text">Questioner Goals</h3>
 			</div>
 		</div>
 	</footer>
-	<script src="../assets/js/index.js"></script>
+	<script src="../scripts/index.js"></script>
 </body>
 
 </html>
```
Comment (Oluwafayokemi) [inline UI/pages/login.html:95]: Do remove all end of line codes

File: server/routes/Meetup.js [hunk 2]
```diff
@@ -2,8 +2,7 @@ import express from 'express';
 import morgan from 'morgan';
 import bodyParser from 'body-parser';
 import Meetup from '../controllers/Meetup';
-import Question from '../controllers/Question';
-import { baseResponse } from '../functions/handlers';
+import Validator from '../helpers/validate';
 
 const router = express.Router();
```
Comment (Oluwafayokemi) [inline server/routes/Meetup.js:4]: Line 7 -13, I don't think this should be parsed here, but in your root file, your app.js/index.js

Comment (Oluwafayokemi) [reply server/routes/Meetup.js:4]: Could there also be another approach to specifying this error other than the morgan parser?

Comment (chikeozulumba) [reply server/routes/Meetup.js:4]: Thank you for your feedback Fayo. 
I have proceeded in removing the morgan dependency, as I already have an error handler implemented for false routes

File: UI/pages/login.html (hunk 1 above)
Comment (chikeozulumba) [reply UI/pages/login.html:95]: Done. Thank you

**Pull request closed without merging**